questionnaires/art_study_end_message/art_study_end_message_armt.json

```diff
@@ -0,0 +1,40 @@
+[
+  {
+    "field_name": "art_study_end_message",
+    "form_name": "art_study_end_message",
+    "section_header": "Thank You For Completing the Questionnaires",
+    "field_type": "info",
+    "field_label": "",
+    "select_choices_or_calculations": [
+      {
+        "label": "Thank you so much for completing Week 10’s questionnaires and cognitive tasks! You have now completed the 10-week study period.",
+        "code": ""
+      },
+      {
+        "label": "We are very grateful for your time and we hope you have enjoyed taking part. A member of the team will be in touch shortly about next steps.",
+        "code": ""
+      },
+      {
+        "label": "If you have any questions before we get in touch, please contact us on 07342 074643 or on ART@kcl.ac.uk.",
```

Comment:
Is it a good idea to make a personal phone number public?

Reply:
Yeah you're right, I will double check with them.

Reply:
I guess this information should also be in the information sheet that was given to participants. Can we say something like "Please contact us using the contact info on the information sheet." Or something similar. The email should be ok to include I guess.